index.js

```diff
@@ -152,22 +178,22 @@ var getFunctionNameSanitized = function () {
 }
 
 var getModuleName = function () {
-  var filename = this.getFileName() || ''
+  var filename = this.getSourceMappedFileName() || ''
   var match = filename.match(MODULE_FOLDER_REGEX)
   if (match) return match[1]
 }
 
 var isApp = function () {
-  return !this.isNode() && !~(this.getFileName() || '').indexOf('node_modules' + path.sep)
+  return !this.isNode() && !~(this.getSourceMappedFileName() || '').indexOf('node_modules' + path.sep)
 }
 
 var isModule = function () {
-  return !!~(this.getFileName() || '').indexOf('node_modules' + path.sep)
+  return !!~(this.getSourceMappedFileName() || '').indexOf('node_modules' + path.sep)
 }
 
 var isNode = function () {
   if (this.isNative()) return true
-  var filename = this.getFileName() || ''
+  var filename = this.getSourceMappedFileName() || ''
```

Comment:
No matter if we are using source maps or not, the node-related call sites should always be detectable using our current approach. So I don't think this is necessary